tools/build_gcsfuse/main.go

```diff
@@ -133,10 +133,10 @@ func buildBinaries(dstDir, srcDir, version string, buildArgs []string) (err erro
 		cmd := exec.Command(
 			"go",
 			"build",
-			"-o",
-			path.Join(dstDir, bin.outputPath),
```

Comment:
are daily tests also failing because of this?

Reply:
This issue visible only for cloud top. On VM it is working fine

Reply:
Daily tests are not failing because they use go version 1.20.4
We'll see this error only after the go version on our test script is updated to 1.21.x

Reply:
Please mention these thing in description.